### ISSUE TITLE
WebVR and Viewpoint interactions

### ISSUE THREAD

**Issue by npolys** [x3dom/x3dom#874]
In the new WebVR branch, when one clicks to go to VR mode
 the runtime binds to a new default Viewpoint.

It would be great if this code found the current Viewpoint properties and copied those ... 
   or just route the WebVR navigation data to the currently bound Viewpoint .

**Comment by andreasplesch** [comment]
The issue may be that current pose of the HMD has to be taken into account when switching.

I think the HMD navigation data - the left and right view matrices - are directly used, perhaps under the current viewpoints local transform. Is it currently possible to have an animated viewpoint in VR ?

I think you are saying that when switching the actual rendered view should not change, to avoid any disruption.

**Comment by npolys** [comment]
Yes... the currently bound viewpoint (and its pos and rot) should be taken into account when processing the headpose data... to avoid disruption ... now when you go to VR mode your view ends up in a strange place
 (and there is no working navigation )

**Comment by npolys** [comment]
I don't see a pattern about where you end up in the scene when you go into VR mode... will have to trace back through the code....

**Comment by npolys** [comment]
Perhaps this is handled here:

https://github.com/x3dom/x3dom/blob/webVR/src/X3DDocument.js

**Comment by npolys** [comment]
I also note that the problem is not apparent when using the Webvr emulator in chrome, but very real on Oculus Go
